File: src/dotnet/Common/Models/Agents/InternalContextAgent.cs
namespace FoundationaLLM.Common.Models.Agents
{
    /// <summary>
    /// The Internal Context agent metadata model.
    /// </summary>
    public class InternalContextAgent : AgentBase
    {
        /// <summary>
        /// Set default property values.
        /// </summary>
        public InternalContextAgent() =>
            Type = AgentTypes.InternalContext;
    }
}

File: src/python/PythonSDK/foundationallm/models/metadata/internal_context_agent.py
"""
Class Name: InternalContextAgent

Description: Encapsulates the metadata for the agent
fulfilling the orchestration request. This agent is used
as a pass-through of the prompt directly to the LLM
"""
from .agent_base import AgentBase

class InternalContextAgent(AgentBase):
    """Internal Context Agent metadata model."""
